console/backend/src/main/java/org/frankframework/management/web/ApiExceptionHandler.java

```diff
@@ -37,96 +40,94 @@
-		return handleSpringException(ex, status, headers);
-	}
-
-	@Override
-	protected ResponseEntity<Object> handleServletRequestBindingException(ServletRequestBindingException ex, HttpHeaders headers, HttpStatusCode status, WebRequest request) {
+@Log4j2
+public class ApiExceptionHandler {
+
+	/** Taken from {@link ResponseEntityExceptionHandler#handleException(Exception, WebRequest)} } */
+	@ExceptionHandler({
+			HttpRequestMethodNotSupportedException.class,
+			HttpMediaTypeNotSupportedException.class,
+			HttpMediaTypeNotAcceptableException.class,
+			MissingPathVariableException.class,
+			MissingServletRequestParameterException.class,
+			MissingServletRequestPartException.class,
+			ServletRequestBindingException.class,
+			MethodArgumentNotValidException.class,
+			NoResourceFoundException.class,
+			AsyncRequestTimeoutException.class,
+			ErrorResponseException.class,
+			MaxUploadSizeExceededException.class,
+
+	})
+	protected final ResponseEntity<Object> handleException(Exception ex, WebRequest request) {
+		if (ex instanceof HttpRequestMethodNotSupportedException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof HttpMediaTypeNotSupportedException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof HttpMediaTypeNotAcceptableException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof MissingPathVariableException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof MissingServletRequestParameterException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof MissingServletRequestPartException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof ServletRequestBindingException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof MethodArgumentNotValidException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof HandlerMethodValidationException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof NoResourceFoundException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof AsyncRequestTimeoutException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
+		} else if (ex instanceof ErrorResponseException subEx) {
+			return handleSpringException(subEx, subEx.getStatusCode(), subEx.getHeaders());
```

Comment:
In plaats van alle implementaties kan je beter kijken naar het interface (ErrorResponse)

Reply:
Looks like all the exceptions listed here implement ErrorResponse somewhere down the inheritance chain, latest commit makes use of th